storybook/pages/WCUriInputPage.qml

```diff
@@ -0,0 +1,76 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import shared.popups.walletconnect.PairWCModal 1.0
+
+import Storybook 1.0
+
+SplitView {
+    id: root
+
+    ColumnLayout {
+        SplitView.fillHeight: true
+        SplitView.fillWidth: true
+
+        WCUriInput {
+            id: wcInput
+            Layout.fillWidth: true
+            Layout.alignment: Qt.AlignCenter
+            Layout.margins: 16
+            // text: "wc:825fbaeb53eeeb08e53a8ddf40cec7996056f49647ab5c39663a2a102920d81c@2?expiryTimestamp=1719495004&relay-protocol=irn&symKey=2eaa97fa11774efb67fd7c93efe92773a7b60650c5cb2621abbdba02cdd4040c"
+        }
+    }
+
+    Pane {
+        Layout.fillWidth: true
+
+        ColumnLayout {
+            TextInput {
+                id: placeHolderInput
+                text: "Input state"
+            }
+            CheckBox {
+                id: pendingCheckBox
+                text: "pending"
+                checked: false
+
+                onCheckedChanged: {
+                    let items = InspectionUtils.findVisualsByTypeName(wcInput, "StatusBaseInput")
+                    if (items.length === 1) {
+                        items[0].pending = pendingCheckBox.checked
+                    }
+                }
+            }
+            CheckBox {
+                id: validCheckBox
+                text: "valid"
+                checked: true
+
+                onCheckedChanged: {
+                    let items = InspectionUtils.findVisualsByTypeName(wcInput, "StatusBaseInput")
+                    if (items.length === 1) {
+                        items[0].valid = validCheckBox.checked
+                    }
+                }
+            }
+        }
+    }
+
+    // When using Component.onCompleted and InspectionUtils.findVisualsByTypeName
+    // needed StatusBaseInput will not be found yet
+    Timer {

```

Comment:
Why do we need this timer? It's not working for me the first time storybook is launched. Please see the attached video:

https://github.com/status-im/status-desktop/assets/47811206/3a7dd628-1b64-40aa-a06f-fa20853c5c44